clevertap-core/src/main/java/com/clevertap/android/sdk/variables/JsonUtil.java

```diff
@@ -50,6 +52,10 @@ public static JSONArray listToJsonArray(Iterable<?> list) throws JSONException {
   }
 
   public static <T> Map<String, T> mapFromJson(JSONObject object) {
+    return mapFromJson(object, false);
+  }
+
+  public static <T> Map<String, T> mapFromJson(JSONObject object, boolean normaliseNames) {
     if (object == null) {
       return null;
     }
```

Comment:
keysIterator is actually of type `Iterator<String>`. If it is changed the cast of `key` to (String) will be redundant.

Reply:
done.